queries.sql

```diff
@@ -79,3 +79,20 @@ SELECT MIN(weight_kg) FROM animals;
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth  BETWEEN '1990-01-01' AND '2000-12-31';
+
+
+-- Write queries (using JOIN) to answer the following questions
+
+SELECT names, full_name FROM animals INNER JOIN owners ON animals.owners_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.names, species.names FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.names = 'Pokemon';
+
+SELECT owners.full_name, animals.names FROM owners LEFT JOIN animals ON owners.id = animals.owners_id;
+
+SELECT COUNT(animals.names), species.names FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.names;
+
+SELECT owners.full_name, animals.names, species.names FROM owners INNER JOIN animals ON owners.id = animals.owners_id INNER JOIN species ON species.id = animals.species_id WHERE species.names = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+
+SELECT owners.full_name, animals.names, animals.escape_attempts FROM owners INNER JOIN animals ON owners.id = animals.owners_id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+
+SELECT owners.full_name, COUNT(*) FROM owners INNER JOIN animals ON owners.id = animals.owners_id GROUP BY owners.full_name ORDER BY count DESC LIMIT 1;
```

Comment:
- [ ] This query is wrong. You are supposed to select from animals table and join the owners table. You can also see so by checking your query results screenshot which is returning a wrong result.  Kindly ake this change.

Reply:
@Nemwel-Boniface  thank you for your time.
![most](https://user-images.githubusercontent.com/105460631/207730629-72f5a3e1-e85b-4c95-8d0b-a99a1e9b1fb3.png)